editions/tw5.com/tiddlers/filters/enlist Operator.tid

```diff
@@ -14,13 +14,12 @@ tags: [[Filter Operators]] [[Field Operators]] [[Selection Constructors]] [[Nega
 title: enlist Operator
 type: text/vnd.tiddlywiki
 
-<<.tip " Literal filter operands cannot contain square brackets but you can work around the issue by using a variable:">>
+<<.tip """Literal filter operands cannot contain square brackets but you can work around the issue by using a variable. ''Learn more at:'' [[SetWidget]] "Filtered List Variable Assignment" """>>
```

Comment:
I think it would be better to make the reference more explicit:

```
... ''Learn more in the [[SetWidget]] documentation under the heading "Filtered List Variable Assignment" """>>
```